schematic/schemas/commands.py

```diff
@@ -151,3 +172,4 @@ def convert(
     # get the execution time
     elapsed_time = time.strftime("%M:%S", time.gmtime(end_time - start_time))
     click.echo(f"Execution time: {elapsed_time} (M:S)")
+    return 0
```

Comment:
You should not hard coded the return code. If the command truly works, then it will just return 0

Reply:
so I tried removing this and got an error during linting `error: Return value expected  [return-value]`

Reply:
Never mind this comment. I think writing return 0 is actually okay like Bryan mentioned.